vendor/github.com/Microsoft/hcsshim/legacy.go

```diff
@@ -121,6 +121,16 @@ func (r *legacyLayerReader) walkUntilCancelled() error {
 		if err != nil {
 			return err
 		}
+
+		// Indirect fix for https://github.com/moby/moby/issues/32838#issuecomment-343610048.
+		// Handle failure from what may be a golang bug in the conversion of
+		// UTF16 to UTF8 in files which are left in the recycle bin. Os.Lstat
+		// which is called by filepath.Walk will fail when a filename contains
```

Comment:
Do we have an issue reported in the Golang repository? They're working on the Go 1.10 release, so perhaps there's still time to get it fixed in time for that

Reply:
It's not clear yet if it's golang or Windows. Being investigating separately.